packages/runtime/container-runtime/src/batchManager.ts

```diff
@@ -25,13 +30,9 @@ export class BatchManager {
     private batchContentSize = 0;
 
     public get length() { return this.pendingBatch.length; }
-    public get limit() { return this.hardLimit; }
+    public get limit() { return this.options.hardLimit; }
```

Comment:
If you go options way, then maybe ditch these methods and just expose Readonly<IBatchManagerOptions> ?

Reply:
I like that